--- ./COPYING.orig	2015-10-22 17:16:01.000000000 -0600
+++ ./COPYING	2015-10-23 08:58:28.369418251 -0600
@@ -29,8 +29,6 @@ Code that has been incorporated into Sub
 naturally be under its own license, and will retain that license.
 
 A known list of such code is maintained here:
-* The python/iso8601 module by Michael Twomey, distributed under an MIT style
-  licence - see python/iso8601/LICENSE for details.
 * The runtests.py and python/subunit/tests/TestUtil.py module are GPL test
   support modules. They are not installed by Subunit - they are only ever
   used on the build machine.  Copyright 2004 Canonical Limited.
--- ./Makefile.in.orig	2015-10-22 17:23:07.000000000 -0600
+++ ./Makefile.in	2015-10-23 08:58:28.371418091 -0600
@@ -598,11 +598,6 @@ EXTRA_DIST = \
 	perl/lib/Subunit.pm \
 	perl/lib/Subunit/Diff.pm \
 	perl/subunit-diff \
-	python/iso8601/LICENSE \
-	python/iso8601/README \
-	python/iso8601/README.subunit \
-	python/iso8601/setup.py \
-	python/iso8601/test_iso8601.py \
 	python/subunit/tests/__init__.py \
 	python/subunit/tests/sample-script.py \
 	python/subunit/tests/sample-two-script.py \
@@ -656,7 +651,6 @@ pkgpython_PYTHON = \
 	python/subunit/chunked.py \
 	python/subunit/details.py \
 	python/subunit/filters.py \
-	python/subunit/iso8601.py \
 	python/subunit/progress_model.py \
 	python/subunit/run.py \
 	python/subunit/v2.py \
--- ./README.rst.orig	2015-10-22 17:15:38.000000000 -0600
+++ ./README.rst	2015-10-23 08:58:28.372418011 -0600
@@ -15,9 +15,6 @@
 
   See the COPYING file for full details on the licensing of Subunit.
 
-  subunit reuses iso8601 by Michael Twomey, distributed under an MIT style
-  licence - see python/iso8601/LICENSE for details.
-
 Subunit
 -------
 
