--- COPYING.orig	2018-03-24 08:22:44.000000000 -0600
+++ COPYING	2018-07-03 15:07:50.241733503 -0600
@@ -29,8 +29,6 @@ Code that has been incorporated into Sub
 naturally be under its own license, and will retain that license.
 
 A known list of such code is maintained here:
-* The python/iso8601 module by Michael Twomey, distributed under an MIT style
-  licence - see python/iso8601/LICENSE for details.
 * The runtests.py and python/subunit/tests/TestUtil.py module are GPL test
   support modules. They are not installed by Subunit - they are only ever
   used on the build machine.  Copyright 2004 Canonical Limited.
--- Makefile.am.orig	2018-03-24 08:22:44.000000000 -0600
+++ Makefile.am	2018-07-03 15:08:08.386499408 -0600
@@ -16,11 +16,6 @@ EXTRA_DIST =  \
 	perl/lib/Subunit.pm \
 	perl/lib/Subunit/Diff.pm \
 	perl/subunit-diff \
-	python/iso8601/LICENSE \
-	python/iso8601/README \
-	python/iso8601/README.subunit \
-	python/iso8601/setup.py \
-	python/iso8601/test_iso8601.py \
 	python/subunit/tests/__init__.py \
 	python/subunit/tests/sample-script.py \
 	python/subunit/tests/sample-two-script.py \
@@ -78,7 +73,6 @@ pkgpython_PYTHON = \
 	python/subunit/chunked.py \
 	python/subunit/details.py \
 	python/subunit/filters.py \
-	python/subunit/iso8601.py \
 	python/subunit/progress_model.py \
 	python/subunit/run.py \
 	python/subunit/v2.py \
--- README.rst.orig	2018-03-24 08:22:44.000000000 -0600
+++ README.rst	2018-07-03 15:08:21.114335195 -0600
@@ -15,9 +15,6 @@
 
   See the COPYING file for full details on the licensing of Subunit.
 
-  subunit reuses iso8601 by Michael Twomey, distributed under an MIT style
-  licence - see python/iso8601/LICENSE for details.
-
 Subunit
 -------
 
